contrib/experiments/interpretation/dutchf3_patch/distributed/train.py

```diff
@@ -197,8 +199,14 @@ def run(*options, cfg=None, local_rank=0, debug=False):
 
     model = torch.nn.parallel.DistributedDataParallel(model, device_ids=[device], find_unused_parameters=True)
 
-    snapshot_duration = epochs_per_cycle * len(train_loader) if not debug else 2 * len(train_loader)
+    snapshot_duration = epochs_per_cycle * len(train_loader) if not debug else 2*len(train_loader)
+
+    snapshot_duration = scheduler_step * len(train_loader)
+    if debug:
+        snapshot_duration = 2
```

Comment:
duplicate? I think the first one is the latest version

Reply:
bad merge - resolved.